From 81ae8f32a97cf9969dcf3355ecd1c2349cf25cf7 Mon Sep 17 00:00:00 2001
From: Yuwei Chen <yuwei.chen@intel.com>
Date: Fri, 24 Jul 2020 09:41:22 +0800
Subject: [PATCH] Test case-E for Driver - String.c file

Description: Change the value of 'Narrow Glyphs' in the MdeModulePkg/Application/UiApp/String.c

Repo: edk2
Signed-off-by: Yuwei Chen <yuwei.chen@intel.com>
---
 MdeModulePkg/Application/UiApp/String.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/MdeModulePkg/Application/UiApp/String.c b/MdeModulePkg/Application/UiApp/String.c
index a4d0c32108..7a32b1b2c9 100644
--- a/MdeModulePkg/Application/UiApp/String.c
+++ b/MdeModulePkg/Application/UiApp/String.c
@@ -65,7 +65,7 @@ FONT_PACK_BIN mFontBin = {
         0x4C,
         0x46,
         0x76,
-        0x72,
+        078,
         0x00,
         0x00,
         0x00,
-- 
2.27.0.windows.1

